README.md

```diff
@@ -13,13 +13,13 @@ ShevyJS takes the concepts of the original [Shevy](https://github.com/kyleshevli
 Shevy is available as a module from npm:
 
 ```
-npm install shevyjs --save-dev
+npm install shevyjs --save
```

Comment:
Think we can even drop `--save` since it defaults to a dependency.

Reply:
Ahh yes, in the olden days of `npm` it didn't